caravel/assets/stylesheets/dashboard.css

```diff
@@ -44,3 +44,28 @@
 .dashboard  div.nvtooltip {
   z-index: 888;  /* this lets tool tips go on top of other slices  */
 }
+
+.add-slice-selected {
+  background: gray;
+  color: white;
+}
+
+.add-slice-selected:hover {
+  background: lightgray !important;
```

Comment:
what class is `!important` overwriting here? is there a way to avoid using !important here?